src/pages/leeszaal/[readingRoomSlug]/[objectId]/index.tsx

```diff
@@ -45,6 +46,7 @@ const ObjectDetailPage: NextPage = () => {
 	const [activeBlade, setActiveBlade] = useState<MediaActions | null>(null);
 	const [mediaType, setMediaType] = useState<MediaTypes>(null);
 	const [pauseMedia, setPauseMedia] = useState(true);
+	const [mediaUrl, setMediaUrl] = useState<string | undefined>(undefined);
```

Comment:
Je hoeft hier niet `undefined` mee te geven `useState()` is hetzelfde als `useState(undefined)`